docs/user-guide.md

```diff
@@ -223,6 +223,66 @@ outputs:
   - file: cluster_data.json
 ```
 
+## Inputs
+
+### GKE API and GKE Local
+
+GKE API input should be enabled for both - cluster configuration verification
+and for scalability checks on clusters which can be accessed online by the tool.
```

Comment:
Lets enable it by default and mention this here.
GKE local can be enabled instead if needed.